tactics/tacticals.ml

```diff
@@ -29,8 +32,10 @@ exception FailError of int * Pp.t Lazy.t
 let catch_failerror (e, info) =
   match e with
   | FailError (lvl,s) when lvl > 0 ->
-    Exninfo.iraise (FailError (lvl - 1, s), info)
-  | e -> Control.check_for_interrupt ()
+    tclZERO ~info (FailError (lvl - 1, s))
+  | e ->
+    Control.check_for_interrupt ();
```

Comment:
Would be interesting to see a bench with these removed, last time was around 3% improv